scripts/test/SANS/gui_logic/run_tab_presenter_test.py

```diff
@@ -75,6 +75,7 @@ def setUp(self):
 
         self.os_patcher = mock.patch('sans.gui_logic.presenter.run_tab_presenter.os')
         self.addCleanup(self.os_patcher.stop)
+        self.os_patcher.start()
```

Comment:
Is this line not doing the same thing as the one below?

Reply:
The tests were failing because we were trying to stop a patcher which hadn't been started. I agree that they do the same thing,  but adding that line made the tests run.

I did mean to sort this out before opening the PR but I forgot